db/migrate/20210705182718_create_topics.rb

```diff
@@ -0,0 +1,8 @@
+class CreateTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :topics do |t|
+      t.string :name, null: false
```

Comment:
`t.string :name, null: false, index: true`

When you think you will use a column for ordering the records, or for doing searches, it is good to consider adding a database index to it.

Be careful and try not to abuse when a model has many columns, to add index to every column or things like that.